_posts/2014-11-21-Meetup-In-Prague.md

```diff
@@ -21,12 +21,12 @@ In the end, Adam, lead developer of [**Dredd**](https://github.com/apiaryio/dred
 
 For those of you who could not attend, we have video of their presentations. 
 
-<iframe width="560" height="315" src="//www.youtube.com/embed/qjadM_6ZN8s" frameborder="0" allowfullscreen></iframe>
+<iframe width="560" height="315" src="//www.youtube-nocookie.com/embed/qjadM_6ZN8s" frameborder="0" allowfullscreen></iframe>
```

Comment:
that video is not accessible anymore?

Reply:
so it probably can be removed?

Reply:
Oh, I did not check that. So yes, that can be removed.